### PR TITLE
Fix binary arguments (clap)

### PR DESCRIPTION
**Pull request by TornaxO7**
# Description

This should make `leftwm-check` working again.

Fixes #(993) (only half)

## Type of change

- [ ] Development change (no change visible to user)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation only update (no change to the factual codebase)
- [ ] This change requires a documentation update

## Updated user documentation:

Not needed

# Checklist:

- [x] Ran `make test-full` locally with no errors or warnings reported
  Note: To fully reproduce CI checks, you will need to run `make test-full-nix`. Usually, this is not neccesary.
- [x] Manual page has been updated accordingly
- [x] Wiki pages have been updated accordingly (to perform **after** merge)

### CODE REVIEW

File: leftwm/src/bin/leftwm-command.rs [hunk 1]
```diff
@@ -12,15 +12,10 @@ async fn main() -> Result<()> {
...
         .arg(
-            arg!("command").help("The command to be sent. See 'list' flag."), // .required(true)
-                                                                              // .multiple(true)
-        )
-        .arg(
-            arg!("list")
-                .help("Print a list of available commands with their arguments.")
-                .short('l')
-                .long("list"),
+            arg!(--"command" [COMMAND] "The command to be sent. See 'list' flag."), // .required(true)
```
Comment (Eskaan) [inline leftwm/src/bin/leftwm-command.rs:15]: This is wrong, we don't want to `leftwm-command --command "Command"` but `leftwm-command "Command"`. 
```suggestion
            arg!(name:command [COMMAND] "The command to be sent. See 'list' flag."),
```

File: leftwm/src/bin/leftwm-check.rs [hunk 2]
```diff
@@ -10,37 +10,41 @@ use std::path::Path;
...
+                .help("Outputs received configuration file.")
         )
         .arg(
-            arg!("migrate")
+            Arg::new(MIGRATE_ARG)
                 .short('m')
                 .long("migrate-toml-to-ron")
-                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually."),
+                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually.")
         )
```
Comment (Eskaan) [inline leftwm/src/bin/leftwm-check.rs:39]: You can use `.args(&[<args>])` to make this more readable. Also in the other files.

**Review by Eskaan — CHANGES_REQUESTED** [review]
Some things I wanted to mention to explain

File: leftwm/src/bin/leftwm-check.rs (hunk 2 above)
Comment (TornaxO7) [reply leftwm/src/bin/leftwm-check.rs]: Can I assume that those problems don't exist anymore due to your PR?

**Pull request merged**